WordPress/Classes/ViewRelated/Gutenberg/Processors/GutenbergBlockProcessor.swift

```diff
@@ -0,0 +1,110 @@
+import Foundation
+import Aztec
+
+/// Struct to represent a Gutenberg element
+///
+public struct GutenbergBlock {
+    public let name: String
+    public let attributes: [String: Any]
+    public let content: String
+}
+
+/// A class that processes a gutenberg post content and replaces the designated gutenberg block for the replacement provided strings.
+///
+public class GutenbergBlockProcessor: Processor {
+
+    /// Whenever an Guntenberg block  is found by the processor, this closure will be executed so that elements can be customized.
```

Comment:
Small typos here

```suggestion
    /// Whenever a Gutenberg block is found by the processor, this closure will be executed so that elements can be customized.
```